distributor-widget/integrations.md

```diff
@@ -119,6 +122,15 @@ If a hotel is selected, information about it is also added to the event. \(Note:
 
 Some events expose additional data layer variables. They are described separately for each event.
 
+##### TrackingConsents
+
+| | Property | Type | Purpose |
+| :--- | :--- | :--- | :--- |
+| `advertising` | boolean | required | Advertising, e.g. used to help advertisers deliver more relevant advertising or to limit how many times you see an ad. |
+| `functional` | boolean | required | Unnecessary functionality, e.g. remembering user choices made in the past. |
+| `necessary` | boolean | required | Necessary functionality. |
+| `performance` | boolean | required | Performance, e.g. collecting information about how user uses a website, data are anonymized. |
```

Comment:
Why do you mark them as code? "``". I do not see any other use like that on this page.

Reply:
My bad habit, I'll fix it